docs/private-ddn/connector-deployment-resource-limits.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Connector Deployment Resource Limits
+sidebar_position: 5
+description:
+  "Understand how to configure resource limits for connectors in Hasura DDN to optimize performance and cost. Learn about CPU, memory, and deployment considerations."
+keywords:
+  - connector deployment
+  - resource limits
+  - cpu allocation
+  - memory allocation
+seoFrontMatterUpdated: true
+---
+
+# Connector Deployment Resource Limits
+
+## Introduction
+
+Connectors in Hasura DDN can request specific resources for their deployment. These include memory and CPU allocation. Understanding these limits ensures optimized performance and cost efficiency.
+
+## Default Resource Allocation
+
+By default, connectors have the following resource allocations:
+
+- **CPU**: 1 vCPU
+- **Memory**: 2GB RAM
+
+## Custom Resource Limits
+
+Resource limits can only be modified for **Private DDN**. For **Public DDN**, the resource limits are fixed to **1 vCPU and 2GB RAM**, and cannot be downsized or upscaled.
+
+### Public DDN
+
+- **Fixed CPU**: 1 vCPU
+- **Fixed RAM**: 2GB
+- CPU and memory cannot be customized for Hasura-provided connectors.
+- The limit and request are always set to the same value.
+
+### Private DDN
+
+- **Minimum CPU**: 1 vCPU
+- **Minimum RAM**: 2GB
+- The minimum and maximum values can be configured per customer.
+
+## Memory and CPU Specifications
+
+- **Memory in bytes**: Memory can be defined in bytes using units such as `k` (kilobytes), `M` (megabytes), and `G` (gigabytes).
+- **Number of CPU cores**: CPU allocation is specified as a decimal value representing the number of vCPUs.
+
+## Defining Resource Limits in `connector.yaml`
+
+Resource limits can be set at both the top level and per region.
+
+```yaml
+title: "Example connector.yaml configuration"
+kind: Connector
+version: v2
+definition:
+  name: my_connector
+  resources:
+    memory: 128M
+    cpu: 0.5
+  regionConfiguration:
+    - region: us-central1
+      resources:
+        memory: 128M
+        cpu: 2
+```
+
+### Resource Limit Overrides
+
+When resource limits are defined at both the top level and the region level, the region-specific values take precedence over the top-level values for that particular region.
+
+For example, if a connector is configured as follows:
+
+```yaml
+definition:
+  resources:
+    memory: 512M
+    cpu: 1
+  regionConfiguration:
+    - region: us-central1
+      resources:
+        memory: 128M
+        cpu: 2
+```
+
+- In `us-central1`, the connector will have **128MB memory and 2 vCPUs**, overriding the top-level `512MB` memory and `1 vCPU` settings.
+- In any other region not explicitly defined, the top-level values (`512MB memory, 1 vCPU`) will apply.
+
+This allows for fine-grained control over resource allocation based on deployment needs.
+
+## Deployment Considerations
+
+- If no region is specified for a connector, it is deployed to a random region.
+- The granularity for CPU allocation is **0.5 to 4 vCPUs** in increments of **0.5**.
```

Comment:
let's remove this because min & max are configurable, and we don't have increments resolution limit 
```
- The granularity for CPU allocation is **0.5 to 4 vCPUs** in increments of **0.5**.
```